xla/stream_executor/rocm/add_i32_kernel.cu.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
```

Comment:
Should we put this into xla/stream_executor/rocm/rocm_helpers.cu.cc instead?

Reply:
This file is compiled only as hsaco so it is not a good fit for rocm_helpers.cu.cc.